pkg/operator/k8sutil/node.go

```diff
@@ -65,14 +65,18 @@ func ValidNode(node v1.Node, placement rookalpha.Placement) (bool, error) {
 	return ValidNodeNoSched(node, placement)
 }
 
-// GetValidNodes returns all nodes that (1) are not cordoned, (2) meet Rook's placement terms, and
-// (3) are ready.
-func GetValidNodes(rookStorage rookalpha.StorageScopeSpec, clientset kubernetes.Interface, placement rookalpha.Placement) []rookalpha.Node {
+// GetValidNodes returns rookNodes, validNodes
+// rookNodes are rookStorage.Nodes updated to better reflect the corresponding k8s objects:
```

Comment:
How about we call these `desiredNodes` and `validDesiredNodes`? I'm getting confused with the more general terms.

Reply:
Agreed, take this chance to rename these to something more obvious.

Reply:
+1

Reply:
Done